OTRExporter/OTRExporter/Main.cpp

```diff
@@ -63,13 +63,29 @@ static void ExporterProgramEnd()
 {
 	if (Globals::Instance->fileMode == ZFileMode::ExtractDirectory)
 	{
-		printf("Generating OTR Archive...\n");
-		otrArchive = Ship::Archive::CreateArchive(otrFileName, 40000);
+    printf("Creating version file...\n");
 
-		for (auto item : files)
-		{
-			auto fileData = item.second;
-			otrArchive->AddFile(item.first, (uintptr_t)fileData.data(), fileData.size());
+    // Get crc from rom
+    std::string romPath = Globals::Instance->baseRomPath.string();
+    std::vector<uint8_t> romData = File::ReadAllBytes(romPath);
+    uint32_t crc = BitConverter::ToUInt32BE(romData, 0x10);
+
+    // Write crc to version file
+    fs::path versionPath("Extract/version");
```

Comment:
Why not just `std::ofstream versionFile("Extract/version", std::ios::out | std::ios::binary);`?

Reply:
I did at first but I figured doing it this way would avoid all potential issues of using `/` vs `\` as  separator. So just to be sure it's portable.